### PR TITLE
Add prefix length to stream message.

### PR DESCRIPTION
**Pull request by arijitAD**
Thank you for contributing to Mina! Please see `CONTRIBUTING.md` if you haven't
yet. In that doc, there are more details around how to start our CI.

Explain your changes here.
Prefix LEB128 encoded message length to message data.

Explain how you tested your changes here.
Added test `TestMplex_SendLargeMessage` and `TestMplex_SendMultipleMessage`

Checklist:

- [x] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [x] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

Closes #0000
Closes #0000

### CODE REVIEW

File: src/app/libp2p_helper/src/libp2p_helper/main.go [hunk 1]
```diff
@@ -1454,3 +1471,38 @@ func main() {
 }
 
 var _ json.Marshaler = (*methodIdx)(nil)
+
+func uint64ToLEB128(in uint64) []byte {
```
Comment (psteckler) [inline src/app/libp2p_helper/src/libp2p_helper/main.go:1475]: this is unsigned LEB128, right?

Comment (arijitAD) [reply src/app/libp2p_helper/src/libp2p_helper/main.go:1475]: Yes

**Comment by nholland94** [comment]
!ci-build-me

**Comment by nholland94** [comment]
@arijitAD Would you mind pulling the latest compatible changes up into this branch? I believe it should fix some of the CI issues that are occurring on this branch atm. Either a rebase or a merge should work.

**Comment by arijitAD** [comment]
Closing. Duplicate PR. https://github.com/MinaProtocol/mina/pull/8721

**Pull request closed without merging**